.circleci/config.yml

```diff
@@ -71,13 +71,54 @@ jobs:
             - "./target"
             - "./pallets/runtime/target"
             - "./pallets/runtime/wasm/target"
+  cli:
+    docker:
+      - image: maxsam4/rust
+    environment:
+      VERBOSE: "1"
+    steps:
+      - checkout
+      - run:
+          name: Build concatenated lock file for caching
+          command: find . -name "Cargo.lock" -not -path "*/target/*" | xargs cat > concatenated.lock
+      - run:
+          name: Store rust version in a file for cache key
+          command: rustc --version > rust.version
+      - restore_cache:
+          keys:
+            - debug-cache-{{ checksum "./rust.version" }}-{{ checksum "./pallets/runtime/Cargo.lock" }}-{{ checksum "./concatenated.lock" }}
+            - debug-cache-{{ checksum "./rust.version" }}-{{ checksum "./pallets/runtime/Cargo.lock" }}
+            - debug-cache-{{ checksum "./rust.version" }}
+            - debug-cache
+      - run:
+          name: Build release
+          command: cargo build -j 1 --release
+          no_output_timeout: 4h
+      - run:
+          name: cli node instance 
+          command: ./target/release/polymesh --dev --pool-limit 100000 -d /tmp/pmesh-primary-node
+          working_directory: ./target/release
```

Comment:
This line is not needed since you are directly using `./target/release/polymesh` above. Alternatively, you can keep this line and use `./polymesh` above.

`working_directory` basically cd's you inside the directory you type.

Reply:
ohh okay cool thanks I'll make the appropriate changes

Reply:
I meant that you can change `command: ./target/release/polymesh` to `command: ./polymesh` if you don't change `working_directory`.

However, the easier method is to simply delete the `working_directory` line and do not change the `command` line.

Reply:
ohh, I  misunderstood alright cool